tests/crypto/README.rst

```diff
@@ -0,0 +1,345 @@
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| EC-JPAKE           | secp256r1   | `J-PAKE: Password-Authenticated Key Exchange by Juggling`_                 | Custom                                                                     |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+.. note::
+   Nordic devices such as nRF51, nRF52810, or nRF52811 cannot run the full test suite because of limited flash capacity.
+   A recommended approach in such case is to run subsets of the tests one by one.
+
+
+Building and running
+********************
+
+.. |test path| replace:: :file:`tests/crypto/`
+
+.. include:: /includes/build_and_run_test.txt
+
+There are multiple ways to build the tests.
+See :ref:`nrf_security_readme` for additional information about configuring of the Nordic Security Module.
+You can use the following configuration files to build it in a specific setup:
```

Comment:
```suggestion
You can use the following configuration files to build a test in a specific setup:
```
Not sure if that is correct or if your talking about the Nordic Security Module here? If you're referring to the tests, you used plural before, so "it" is wrong.